docs/devGuide/writingComponents.md

```diff
@@ -0,0 +1,115 @@
+{% set title = "Writing Components" %}
+<span id="title" class="d-none">{{ title }}</span>
+
+<frontmatter>
+  title: "{{ title }}"
+  layout: devGuide.md
+  pageNav: default
+</frontmatter>
+
+# {{ title }}
+
+<div class="lead">
+
+This page explains various concerns related to MarkBind components, focused on implementation and testing.
+</div>
+
+## Implementing Components
+
+There are multiple ways to implement MarkBind components.
+
+<box type='warning' light>
```

Comment:
I just noticed that most of the devGuide uses the seamless style for boxes! I think it might be good to use them on this page too, just for visual consistency